pynest/examples/clopath_synapse_spike_pairing.py

```diff
@@ -46,56 +46,59 @@
-              'A_LTD': 14.0e-5,
-              'A_LTP': 8.0e-5,
-              'tau_u_bar_minus': 10.0,
-              'tau_u_bar_plus': 7.0,
-              'delay_u_bars': 4.0,
-              'a': 4.0,
-              'b': 0.0805,
-              'V_reset': -70.6 + 21.0,
-              'V_clamp': 33.0,
-              't_clamp': 2.0,
-              't_ref': 0.0,
-              }
+nrn_params = {
+    "V_m": -70.6,
+    "E_L": -70.6,
+    "C_m": 281.0,
+    "theta_minus": -70.6,
+    "theta_plus": -45.3,
+    "A_LTD": 14.0e-5,
+    "A_LTP": 8.0e-5,
+    "tau_u_bar_minus": 10.0,
+    "tau_u_bar_plus": 7.0,
+    "delay_u_bars": 4.0,
+    "a": 4.0,
+    "b": 0.0805,
+    "V_reset": -70.6 + 21.0,
+    "V_clamp": 33.0,
+    "t_clamp": 2.0,
+    "t_ref": 0.0,
+}
 
 
 ##############################################################################
 # Hardcoded spike times of presynaptic spike generator
 
 spike_times_pre = [
     # Presynaptic spike before the postsynaptic
-    [ 20.0,  120.0,  220.0,  320.0,  420.0],          # noqa
-    [ 20.0,   70.0,  120.0,  170.0,  220.0],          # noqa
-    [ 20.0,   53.3,   86.7,  120.0,  153.3],          # noqa
-    [ 20.0,   45.0,   70.0,   95.0,  120.0],          # noqa
-    [ 20.0,   40.0,   60.0,   80.0,  100.0],          # noqa
+    [20.0, 120.0, 220.0, 320.0, 420.0],  # noqa
+    [20.0, 70.0, 120.0, 170.0, 220.0],  # noqa
+    [20.0, 53.3, 86.7, 120.0, 153.3],  # noqa
+    [20.0, 45.0, 70.0, 95.0, 120.0],  # noqa
+    [20.0, 40.0, 60.0, 80.0, 100.0],  # noqa
     # Presynaptic spike after the postsynaptic
-    [120.0,  220.0,  320.0,  420.0,  520.0,  620.0],  # noqa
-    [ 70.0,  120.0,  170.0,  220.0,  270.0,  320.0],  # noqa
-    [ 53.3,   86.6,  120.0,  153.3,  186.6,  220.0],  # noqa
-    [ 45.0,   70.0,   95.0,  120.0,  145.0,  170.0],  # noqa
-    [ 40.0,   60.0,   80.0,  100.0,  120.0,  140.0]]  # noqa
+    [120.0, 220.0, 320.0, 420.0, 520.0, 620.0],  # noqa
+    [70.0, 120.0, 170.0, 220.0, 270.0, 320.0],  # noqa
+    [53.3, 86.6, 120.0, 153.3, 186.6, 220.0],  # noqa
+    [45.0, 70.0, 95.0, 120.0, 145.0, 170.0],  # noqa
+    [40.0, 60.0, 80.0, 100.0, 120.0, 140.0],
+]  # noqa
 
 ##############################################################################
 # Hardcoded spike times of postsynaptic spike generator
 
 spike_times_post = [
-    [ 10.0,  110.0,  210.0,  310.0,  410.0],          # noqa
-    [ 10.0,   60.0,  110.0,  160.0,  210.0],          # noqa
-    [ 10.0,   43.3,   76.7,  110.0,  143.3],          # noqa
-    [ 10.0,   35.0,   60.0,   85.0,  110.0],          # noqa
-    [ 10.0,   30.0,   50.0,   70.0,   90.0],          # noqa
-    [130.0,  230.0,  330.0,  430.0,  530.0,  630.0],  # noqa
-    [ 80.0,  130.0,  180.0,  230.0,  280.0,  330.0],  # noqa
-    [ 63.3,   96.6,  130.0,  163.3,  196.6,  230.0],  # noqa
-    [ 55.0,   80.0,  105.0,  130.0,  155.0,  180.0],  # noqa
-    [ 50.0,   70.0,   90.0,  110.0,  130.0,  150.0]]  # noqa
+    [10.0, 110.0, 210.0, 310.0, 410.0],  # noqa
```

Comment:
The `# noqa` seems to have no effect here and the table formatting is destroyed, similarly above. Can this be avoided?

Reply:
same as above